Computer_vision_workloads/Semantic_segmentation/Scaling_batch_inference.ipynb

```diff
@@ -2124,29 +2127,56 @@
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "0d8a0b7a-3cff-4aea-9b92-c28200560728",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4e512ca",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "### Shutdown Ray runtime"
+    "### YOUR CODE HERE ###"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "fc3aae2c-491f-41a9-bd67-09395f398890",
+   "cell_type": "markdown",
+   "id": "fececa99",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "ray.shutdown()"
+    "**Solution**"
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "72cdc4c3-5c2e-489a-9382-d8ca46a26a1f",
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14709a03",
    "metadata": {},
+   "outputs": [],
    "source": [
-    "Disconnect the worker, and terminate processes started by `ray.init()`."
+    "### SAMPLE IMPLEMENTATION ###\n",
+    "\n",
+    "from transformers import AutoConfig, AutoModelForSemanticSegmentation, AutoFeatureExtractor\n",
```

Comment:
These are same as in part 3 (sequential implementation). Let's use imports from there

Reply:
Instead of using `AutoConfig, AutoModelForSemanticSegmentation, AutoFeatureExtractor`, the new implementation now uses the same `SegformerForSemanticSegmentation` and `SegformerFeatureExtractor` from the beginning of the module.